### ISSUE TITLE
Visualize children of friend entries.

### ISSUE THREAD

**Issue by poulpoulsen** [Kungsgeten/org-brain#101]
Hello, i want to visualize more than one files in one view. All files have headlines. I want to create a view with the files as friends, horizontal aligned.
then i want connect headlines between the files as connect one, or as parent with children from File a and/or b.
Is it possible?

Regards
Poul

**Comment by Kungsgeten** [comment]
Hi!

No, I'm afraid it isn't possible to visualize several entries at the same time. At the time being `org-brain-visualize` is built to have a single `*org-brain*` buffer. It should be possible to add support for multiple `org-brain` buffers, and in that case I guess there could be functionality for communicating between the open buffers (similar to `dired`).

Supporting several `org-brain` buffers would require (I think) a lot of changes to the code, and isn't something I myself plan to add in the near future. If anyone else would like to try and implement it, I'm all for that.

**Comment by poulpoulsen** [comment]
Hello, thanks for the reply.
I will try to write a little bit clearer what i want.
Szenario:
I load File A into org-brain-visualize.
Headline are showing.
Now i put cursor on filename and press f for friendship.
now i load the desired file and it shows up as friendship.
My wish:
the desired file should also show all headlines in it.
Then i could add connections.
And it's viewed in the actual org-brain buffer, not a second one.

Is it clearer now, what i want?

Regards
Poul

**Comment by Kungsgeten** [comment]
If I understand correctly you'd like to see the children of friends to the current entry? Similar to how siblings work (other children of the entry's parents), but for friends? That would probably be doable, but would consume quite a lot more screen estate.

**Comment by poulpoulsen** [comment]
Yes, that is what i mean. With this visualization i could see and create connections out of different org headings from different org files.

**Comment by Kungsgeten** [comment]
This is a feature which I think would take quite a while to code. I won't try to implement it myself, but if anyone else would like to submit a pull request (keeping the code relatively clean) I'll be happy to discuss it.
